cmd/templatizer/app/renderer/render.go

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2021 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package renderer
+
+import (
+	"strconv"
+
+	"github.com/jaegertracing/jaeger/cmd/templatizer/app"
+	esTemplate "github.com/jaegertracing/jaeger/pkg/es"
+	"github.com/jaegertracing/jaeger/plugin/storage/es"
+)
+
+var fixMappingFunc = es.FixMapping
+var loadMappingFunc = es.LoadMapping
+var mapping string
+var err error
```

Comment:
It looks like these stem from needing to mock out the function's behaviour. You've already done the hard work of creating interfaces and mocks for these, why not use them?

I suggested removing these and please refer to my suggestions.